src/UploadImageBehavior.php

```diff
@@ -127,6 +131,10 @@ protected function afterUpload()
     protected function createThumbs()
     {
         $path = $this->getUploadPath($this->attribute);
+        if (!is_file($path)) {
```

Comment:
I do not understand why this is necessary?

Reply:
In case when createThumbsOnRequest == true and original file is already deleted. This is not necessary, but if for some reason thumb file was deleted from disc, later generateImageThumb with will not execute and there will no fatal php error, only no images will be created